tests/enterprise-contract/contract.go

```diff
@@ -347,8 +347,7 @@ var _ = framework.EnterpriseContractSuiteDescribe("Enterprise Contract E2E tests
 						defaultECP.Spec, ecp.SourceConfig{Include: []string{"minimal"}})
 					Expect(fwk.AsKubeAdmin.TektonController.CreateOrUpdatePolicyConfiguration(namespace, policy)).To(Succeed())
 
-					generator.WithComponentImage("quay.io/redhat-appstudio/ec-golden-image:e2e-test-out-of-date-task")
-					generator.AppendComponentImage("quay.io/redhat-appstudio/ec-golden-image:e2e-test-unacceptable-task")
+					generator.WithComponentImage("quay.io/redhat-appstudio/ec-golden-image:latest")
```

Comment:
This test scenario needs more than 2 images,  would it make sense just update the WithComponentImage and keep the AppendComponentImage there?

Reply:
sure, done